drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -312,13 +317,42 @@ function DragAndDropTemplates(configuration) {
         )
     };
 
-    var mainTemplate = function(ctx) {
-        var problemTitle = ctx.show_title ? h('h2.problem-title', {innerHTML: ctx.title_html}) : null;
-        var problemHeader = ctx.show_problem_header ? h('h3.title1', gettext('Problem')) : null;
+    var itemFeedbackPopupTemplate = function(ctx) {
         var popupSelector = 'div.popup';
-        if (ctx.popup_html && !ctx.last_action_correct) {
+        var msgs = (ctx.feedback_messages || []).filter(function(msg) { return (msg) ? true : false; });
+        var popup_content;
+
+        if (msgs.length > 0 && !ctx.last_action_correct) {
             popupSelector += '.popup-incorrect';
         }
+
+        if (msgs.length > 1) {
+            popup_content = h("div.popup-content", {}, [
+                (!ctx.last_action_correct) ? h("p", {}, gettext("Some of your answers were not correct")) : null,
+                h("p", {}, gettext("Hints:")),
+                h("ul", {}, msgs.map(function(message) {
+                    return h("li", {innerHTML: message}, []);
+                }))
+            ])
+        } else {
```

Comment:
@itsjeyd no, we didn't. This would be relatively simple to address, so I would suggest to keep this as is now, and draw upstream reviewers attention to this feature - than if they think it should behave differently, I'll replace it.